src/graphql/main/schema.graphql

```diff
@@ -929,7 +934,8 @@ input PriceInput {
 
 interface PriceInterface {
   base: SafeInt!
-  currencyUnit: String!
+  currencyUnit: CurrencyUnit!
+  minorUnitToMajorUnitOffset: Int!
```

Comment:
I like this name. It's longer than something like `currencyMinorUnitOffset` but it removes all ambiguity.